src/resources/resources.json

```diff
@@ -603,6 +603,32 @@
       "redirectMessage": "Sie werden in {seconds, plural, =1 {# Sekunde} other {# Sekunden}} auf die <0>Anmeldeseite</0> weitergeleitet."
     }
   },
+  {
+    "namespace": "roomMembershipConfirmation",
+    "language": "en",
+    "resources": {
+      "confirmationSuccess": "Congratulations! You are now a member of room <0>{roomName}</0>.",
+      "invitationConfirmation": "Please click on the button in order to join room <0>{roomName}</0>.",
+      "invitationConfirmationDisclaimer": "By clicking on the button you give your consent to ???",
+      "joinButton": "Join room",
+      "invalidOrExpiredInvitationCode": "The invitation code is invalid or expired.",
+      "homeLink": "Home",
+      "redirectMessage": "You will be redirected to the <0>room</0> in {seconds, plural, =1 {# second} other {# seconds}}."
```

Comment:
Maybe "You will be redirected to room <0>room</0> in {seconds, plural, =1 {# second} other {# seconds}}." would sound better?

Reply:
True, English<->German concundrum here. I'll change it.

Reply:
Fixed!